# Image Request Service Name (To request an image from AR Glass Driver)
image_request_service: "AR_Take_Image"
# Image publisher topic Name (AR Glass Driver receives images through this topic)
image_publisher_topic: "AR_Send_Image"